app/policies/user_policy.rb

```diff
@@ -21,6 +21,10 @@ def update_intent?
     show?
   end
 
+  def view_metric_reports?
```

Comment:
I think that this belongs in the Draw policy, not the user policy (all reports are scoped to a Draw, right?)